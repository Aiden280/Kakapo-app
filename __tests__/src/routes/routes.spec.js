import routes from 'routes/';

test('[routes]', () => {
  expect(routes).toMatchSnapshot();
});
