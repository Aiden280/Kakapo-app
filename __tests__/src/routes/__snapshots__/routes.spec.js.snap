// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[routes] 1`] = `
Array [
  Object {
    "component": [Function],
    "path": "/",
    "routes": Array [
      Object {
        "component": [Function],
        "exact": true,
        "path": "/",
      },
      Object {
        "component": [Function],
        "path": "/settings",
        "routes": Array [
          Object {
            "component": [Function],
            "path": "/theme(/:slotNo)",
          },
        ],
      },
      Object {
        "component": [Function],
        "path": "/kakapo",
      },
      Object {
        "component": [Function],
        "path": "/youtube",
      },
      Object {
        "component": [Function],
        "path": "/soundcloud",
      },
      Object {
        "component": [Function],
        "path": "/custom",
      },
      Object {
        "component": [Function],
        "path": "/playlist(/:playlistId)",
      },
      Object {
        "component": [Function],
        "path": "/playlist",
      },
      Object {
        "component": [Function],
        "path": "/share-playlist(/:shareId)",
      },
    ],
  },
]
`;
